site2/docs/getting-started-standalone.md

```diff
@@ -104,34 +104,34 @@ bin/pulsar-client consume my-topic -s 'my-subscription' -p Earliest -n 0
 
 Earliest means consuming from the earliest **unconsumed** message. `-n` configures the number of messages to consume, 0 means to consume forever.
 
-As before, this is useful for trialling things on the command line, but in practice you'll use the Consumer API in your application code, or Pulsar IO connectors for reading data from Pulsar to push to other systems.
+As before, this is useful for experimenting with messages, but in practice you'll use the Consumer API in your application code, or Pulsar IO connectors for reading data from Pulsar to push to other systems.
 
-You'll see the messages that you produce in the previous step:
+You'll see the messages you produce in the previous step:
 
 ```text
 ----- got message -----
 key:[null], properties:[], content:Hello Pulsar!
 ```
 
-## Step 6. Write some more messages
+## Write some more messages
 
 Leave the consume command from the previous step running. If you've already closed it, just re-run it.
 
-Now open a new terminal window and produce more messages, the default message separator is `,`:
+Now open a new terminal window and produce more messages. The default message separator is `,`:
 
 ```bash
 bin/pulsar-client produce my-topic --messages "$(seq -s, -f 'Message NO.%g' -t '\n' 1 10)"
 ```
 
 Note how they are displayed almost instantaneously in the consumer terminal.
 
-## Step 7. Stop the Pulsar cluster
+## Stop the Pulsar cluster
 
 Once you've finished you can shut down the Pulsar cluster. Press **Ctrl-C** in the terminal window in which you started the cluster.
 
-## Further readings
+## Related Topics
```

Comment:
```suggestion
## Related topics
```

Keep consistent for headings (sentence style)

Reply:
Agreed.